python/tank/authentication/login_dialog.py

```diff
@@ -52,6 +54,18 @@
 THREAD_WAIT_TIMEOUT_MS = 5000
 
 
+def _is_context_desktop():
```

Comment:
I'd avoid the word context here, since Context already means something specific in Desktop. Maybe `_is_running_in_desktop()`?